### PR TITLE
PNI Consumer Creep-O-Meter Page Hero

### PR DESCRIPTION
**Pull request by mmmavis**
This is part of the PNI Annual Consumer Creep-O-Meter page work.

- Related PRs/issues: #11094
- Review app test page: https://foundation-s-11094-pni--mn2xnf.mofostaging.net/en/privacynotincluded/articles/annual-consumer-creep-o-meter/
- Localhost test page: `/privacynotincluded/articles/annual-consumer-creep-o-meter/`

---

Note that the animated eyes effect will be added in another ticket.

I also noticed there's some weird alignment issue with the existing PNI nav. Since it's outside of the scope of the consumer creep-o-meter page, I didn't fix it in this PR.

### CODE REVIEW

**Review by tessheinricks — CHANGES_REQUESTED** [review]
Looks great Mavis! 

For mobile, are you able to:
- Revise Creep-O-Meter rating speech bubble to have corner radius of 8 instead of 16
- Revise the vertical spacing between the asterisk in the top right and the header copy to be 8 [M_Vertical-Gap_Quaternary]  

Thank you!

**Comment by mmmavis** [comment]
Hi @tessheinricks , thanks for reviewing. I've made the styling tweaks and the page is ready for review again!

**Review by tessheinricks — CHANGES_REQUESTED** [review]
Thank you for those revisions Mavis! 

I am wondering if there is any way that the asterisk placement on desktop can more closely reflect the figma file (with the illustration bleeding past where the screen ends on the right), which will give a bit more space to the headline?

Please let me know if you have any questions, or if this would present challenges timeline or development wise. Thanks again!

**Comment by mmmavis** [comment]
Hi @tessheinricks 

> I am wondering if there is any way that the asterisk placement on desktop can more closely reflect the figma file (with the illustration bleeding past where the screen ends on the right), which will give a bit more space to the headline?

Currently, it's bleeding outside of the "container:

image

If we want to place the illustration further right to so it bleeds past the viewport I can def do it. However, this means there will be huge gap on larger screen sizes. e.g., 

image

Let me know which you prefer and I'll adjust the code accordingly. Thanks!

**Comment by tessheinricks** [comment]
@mmmavis okay amazing thank you for clarifying! I think leaving it how it is now (bleeding outside of the Content Container) is a better solution than the gap! 

If possible, it might be helpful to extend the horizontal spacing gap between the headline and the asterisk to 40 but again, if there is not enough time, no worries!

**Comment by mmmavis** [comment]
> If possible, it might be helpful to extend the horizontal spacing gap between the headline and the asterisk to 40 but again, if there is not enough time, no worries!

Hi @tessheinricks , we'll have to address this at a later stage. I just tested it out with 40px gap but the illustration would become too cut off to be recognized (in my opinion).

https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/2bafc6dd-f845-46c5-94e3-bf93a7b7f76a

**Comment by tessheinricks** [comment]
@mmmavis Agreed! Thank you for taking the time to test! We can keep it at the 16px [D_Horizontal-Gap_Primary] variable value for now. 👍

**Review by tessheinricks — APPROVED** [review]
Development looks great!

Design team is keeping additional refinement notes in a separate [word doc](https://docs.google.com/document/d/1LoehvJXp3yIyMM39IDThNIA2attLj8gfo5JomwmCN2g/edit?usp=sharing) for final adjustments.

File: network-api/networkapi/wagtailpages/migrations/0105_consumercreepometerpage.py [hunk 1]
```diff
@@ -1,30 +1,64 @@
 # Generated by Django 3.2.21 on 2023-09-22 17:40
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/migrations/0105_consumercreepometerpage.py:1]: linting fixes

**Review by danielfmiranda — APPROVED** [review]
Looks great! Thank you Mavis 🙌 approved!

**Pull request merged**